react-app/vite.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import viteTsconfigPaths from 'vite-tsconfig-paths';
+import eslint from 'vite-plugin-eslint';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    react(),
+    viteTsconfigPaths(),
+    eslint({
+      fix: true,
+      lintOnStart: false,
+    }),
+  ],
+  server: {
+    port: 3000,
```

Comment:
I wish we could change the port using an env var (e.g. `ROMCAL_APP_PORT` or whatever env var name you prefer).

Reply:
asbefore, will do